### ISSUE TITLE
Feature: It would be nice to able to subscribe to the connection state

### ISSUE THREAD

**Issue by JensGabe** [digitalpetri/ethernet-ip#52]

**Comment by kevinherron** [comment]
I agree. I need to plumb this ability into the [underlying library that manages the Channel](https://github.com/digitalpetri/netty-channel-fsm/tree/master/src/main/java/com/digitalpetri/netty/fsm) and then expose it here.

**Comment by WimVriend** [comment]
Any idea when this feature is going to be implemented? 

We also currently experience communication 'overload', when the PLC-program is downloaded from RsLogix. It would be nice to ignore all requests for data from the PLC, until we know that it is ONLINE again.

**Comment by kevinherron** [comment]
I'll try to do this soon, it's been implemented in the library I mentioned at this point.

**Comment by WimVriend** [comment]
That would be great: thanks in advance ;)
